doc/esmvalcore/preprocessor.rst

```diff
@@ -84,6 +84,45 @@ the ozone 3D field. In this case, a derivation function is provided to
 vertically integrate the ozone and obtain total column ozone for direct
 comparison with the observations.
 
+All derivations scripts are located in
```

Comment:
Preferably this should go to a separate chapter: this document describes how to use/read the recipe format, it is not a developer guide. Maybe you can simply move it to a new chapter here, similar to the chapter on fixes?